package.json

```diff
@@ -59,15 +59,15 @@
     "xss": "^1.0.14"
   },
   "devDependencies": {
-    "@nuxt/module-builder": "latest",
-    "@nuxt/schema": "^3.7.3",
-    "@nuxt/test-utils": "^3.7.3",
-    "@nuxtjs/eslint-config-typescript": "latest",
-    "@types/node": "^18.14.4",
-    "eslint": "latest",
-    "nuxt": "^3.7.3",
-    "typescript": "5.2.2",
-    "vitest": "^0.28.5"
+    "@nuxt/module-builder": "^0.5.2",
+    "@nuxt/schema": "^3.7.4",
+    "@nuxt/test-utils": "^3.7.4",
+    "@nuxt/eslint-config": "^0.2.0",
+    "@types/node": "^18.18.1",
+    "eslint": "^8.50.0",
+    "nuxt": "^3.7.4",
```

Comment:
suggestion: Nuxt recently released 3.8 version. Maybe we could migrate to that?

Reply:
ok

Reply:
3.8 requires modification to type imports
See https://github.com/nuxt/nuxt/releases/tag/v3.8.0
Only minor modifications are required in our codebase, I will include these as well